docs-src/zdm-core/modules/migrate/pages/manage-proxy-instances.adoc

```diff
@@ -247,6 +247,12 @@ The first option requires that you deploy a new {zdm-proxy} cluster on the side,
 
 The second option consists of changing the topology of an existing ZDM proxy deployment. For example, let's say that you wish to add three new nodes to an existing six-node deployment. To do this, you need to amend the inventory file so that it contains one line for each machine where you want a proxy instance to be deployed (in this case, the amended inventory file will contain nine proxy IPs, six of which were already there plus the three new ones) and then run the `deploy_zdm_proxy.yml` playbook again. This will stop the existing six proxies, destroy them, create a new nine-node deployment from scratch based on the amended inventory and start it up, therefore resulting in a brief interruption of availability of the whole {zdm-proxy} deployment.
 
+[NOTE]
+====
+{zdm-proxy} containers can be be scaled out by any number of proxies as you see fit and does not need to in multiples of three.
```

Comment:
I would suggest a slight rewording:

`{zdm-proxy} containers can be be scaled out by any number of proxies as you see fit, not necessarily in multiples of three.`